snapcraft/plugins/cmake.py

```diff
@@ -71,7 +71,8 @@ def build(self):
 
         env = self._build_environment()
 
-        self.run(['cmake', sourcedir, '-DCMAKE_INSTALL_PREFIX='] +
+        self.run(['cmake', sourcedir, '-DCMAKE_INSTALL_PREFIX=',
+                  '-DCMAKE_BUILD_TYPE=None', '-DCMAKE_VERBOSE_MAKEFILE=ON'] +
```

Comment:
Can you tie this latter option to --debug. Look at the kernel plug in branch/PR for inspiration.

The former should probably be a plugin option defaulting to None